content/en/docs/reference/glossary/device-plugin.md

```diff
@@ -4,14 +4,22 @@ id: device-plugin
 date: 2019-02-02
 full_link: /docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/
 short_description: >
-  Containers running in Kubernetes that provide access to a vendor specific resource.
+  Pods running in Kubernetes that provide access to a node-local resource.
```

Comment:
They don't even need to be Pods, though, right? I mean, they could just as easily be tiny processes managed by a systemd unit, a standard OS package, or even a shell script running a process via a cron entry, as long as the plugin registers itself with kubelet via /var/lib/kubelet/device-plugins/kubelet.sock. So, perhaps it's better to say something to the effect of:

`Software that runs on a worker node that provides Pods with access to a node-local resource`